src/styles/utils/_media.scss

```diff
@@ -0,0 +1,17 @@
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
```

Comment:
It's a bad practice to split media queries from blocks
How can I understand if there any changes on different breakpoints from `catalog` block file?

Reply:
Please fix it